app/src/main/java/org/mozilla/focus/fragment/HomeFragment.kt

```diff
@@ -63,6 +74,22 @@ class HomeFragment : Fragment() {
         settingsButton.setOnClickListener { v ->
             onSettingsPressed?.invoke()
         }
+
+        registerForContextMenu(view)
+    }
+
+    override fun onCreateContextMenu(menu: ContextMenu?, v: View?, menuInfo: ContextMenu.ContextMenuInfo?) {
+        super.onCreateContextMenu(menu, v, menuInfo)
+//        menu?.setHeaderTitle("Select Option")
+//        menu?.add(Menu.NONE, v!!.getId(), 0, "Remove tile")
+        activity.menuInflater.inflate(R.menu.menu_context_home, menu)
```

Comment:
Nit: rename this because it's not just a home context menu, it's menu_context_hometile